### PR TITLE
Use publish-to-galaxy-plain for collections in ansible-collections/community.*, with explicit list of exceptions

### PR DESCRIPTION
**Pull request by felixfontein**
Make sure that only the `publish-to-galaxy-plain` template (https://github.com/felixfontein/ansible-zuul-jobs/commit/0495329eda10728e0928c990992ab21021a27f70) is used for the set of collections, instead of Zuul merging both the `publish-to-galaxy-plain` and `publish-to-galaxy-3pci` templates for them.

### CODE REVIEW

**Comment by felixfontein** [comment]
Darn, Zuul uses Google's RE2 (https://pypi.org/project/google-re2/) instead of Python `re`, which doesn't support negative lookaheads:

> It uses RE2 under the hood, of course, so various PCRE features (e.g. backreferences, look-around assertions) are not supported.

The syntax is described in https://github.com/google/re2/wiki/Syntax, basically everything that's half-way interesting for this use-case is "NOT SUPPORTED" :crying_cat_face:

**Comment by felixfontein** [comment]
Ok, this is now the solution I really wanted to avoid, but it seems that Zuul really, REALLY wants us to do this :-( I hope it's ok...

**Comment by felixfontein** [comment]
CC @jillr

**Comment by jillr** [comment]
sorry @felixfontein just saw this, will take a look tomorrow!

**Review by jillr — APPROVED** [review]
lgtm, I'm going to get a second review from the team before merging this one just to be safe this time

**Review by alinabuzachis — APPROVED** [review]

**Review by gravesm — APPROVED** [review]

**Comment by felixfontein** [comment]
@jillr @alinabuzachis @gravesm thanks a lot for reviewing and merging! It also looks like this finally works, there's no more third-party check in https://github.com/ansible-collections/community.general/pull/8380 :tada:

**Pull request merged**
